gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -283,7 +283,7 @@ func (vm *VMKeeper) Call(ctx sdk.Context, msg MsgCall) (res string, err error) {
 		m.Release()
 	}()
 	rtvs := m.Eval(xn)
-	ctx.Logger().Info("CPUCYCLES call: ", m.Cycles)
+	ctx.Logger().Info("CPUCYCLES call: ", "num-cycles", m.Cycles)
```

Comment:
```suggestion
	ctx.Logger().Info("CPUCYCLES call", "num-cycles", m.Cycles)
```

If I'm not wrong, we should do this now.

Reply:
You're right, the `:` is unnecessary now:

Applied the suggestion:
[d66b6f7](https://github.com/gnolang/gno/pull/1302/commits/d66b6f70acb1df63cd6f9063ff2fc2b9746d39e0)